packages/config/config/devices/0x0438/touch-thermostat-16a.json

```diff
@@ -0,0 +1,348 @@
+{
+	"manufacturer": "Namron",
+	"manufacturerId": "0x0438",
+	"label": "4512745",
+	"description": "In-Wall Touch Thermostat 16A",
+	"devices": [
+		{
+			"productType": "0x0200",
+			"productId": "0xd025"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"paramInformation": [
+		{
+			"#": "2",
+			"label": "Over-Current Threshold",
+			"description": "When detected current is above threshold, relay turns off and sends notification. Allowable range: 0, 5-16",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 16,
+			"unit": "A",
+			"defaultValue": 16
+		},
+		{
+			"#": "3",
+			"label": "Buzzer Vibration Level",
+			"description": "Set buzzer and vibration level.",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 2,
+			"defaultValue": 1,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disable",
+					"value": 0
+				},
+				{
+					"label": "Low",
+					"value": 1
+				},
+				{
+					"label": "High",
+					"value": 2
+				}
+			]
+		},
+		{
+			"#": "4",
+			"label": "Powerloss Recover Settings",
+			"description": "Recover settings after power loss / power reset.",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 1,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disable",
+					"value": 0
+				},
+				{
+					"label": "Enable",
+					"value": 1
+				}
+			]
```

Comment:
Please replace with an `$import` of https://github.com/zwave-js/node-zwave-js/blob/a23bffff500afa6cbe8299f8b3c775b1bf18f771/packages/config/config/devices/templates/master_template.json#L367